frontend/src/components/VideoCall/VideoFrontend/components/ParticipantList/ParticipantList.tsx

```diff
@@ -1,56 +1,56 @@
-import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
```

Comment:
Why did we switch from single to double quotes?

Reply:
I must've done some command for reformatting the file at some point, should be fixed now